pallets/dkg-proposal-handler/src/lib.rs

```diff
@@ -472,6 +472,25 @@ pub mod pallet {
 				Err(Error::<T>::ProposalMustBeUnsigned.into())
 			}
 		}
+
+		/// Force remove an unsigned proposal from the queue
+		#[pallet::weight(<T as Config>::WeightInfo::force_remove_unsigned_proposal())]
+		#[pallet::call_index(2)]
+		pub fn force_remove_unsigned_proposal(
+			origin: OriginFor<T>,
+			typed_chain_id: TypedChainId,
+			key: DKGPayloadKey,
+		) -> DispatchResultWithPostInfo {
+			// Call must come from root (likely from a democracy proposal passing)
+			<T as pallet::Config>::ForceOrigin::ensure_origin(origin)?;
+			UnsignedProposalQueue::<T>::remove(typed_chain_id, key);
```

Comment:
Does `remove` return a value? maybe if it does not exist, we return an error, instead of failing silently.

Reply:
It does not, will add a check here